shub/config.py

```diff
@@ -131,6 +133,8 @@ def save(self, path=None):
             yml['apikeys'] = self.apikeys
             yml['version'] = self.version
             yml['stacks'] = self.stacks
+            if self.eggs:
+                yml['requirements'] = {'eggs': self.eggs}
```

Comment:
Makes sense to do it in another PR. I'll add one in a bit. Or do you think it should also be in a later release?